### DISCUSSION TITLE
How to use normal mode in create dialog

### DISCUSSION THREAD

**Discussion by yangyagami** [nvim-tree/nvim-tree.lua#2188] [Q&A]
I can't use <esc> to enter normal mode in create dialog, only can use mouse or arrow key to move.

**Comment by alex-courtis** [comment]
Unfortunately that is a limitation of `:h vim.ui.input()`, `:h input`

Command line editing `<c-f>` is not supported.

You can use a plugin such as [dressing](https://github.com/stevearc/dressing.nvim) to enhance your inputs.
